plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.properties

```diff
@@ -12,3 +12,4 @@ deployFlexMenuLabel=Deploy to App Engine Flexible...
 deployFlexExistingWarJarCommandDescription=Uploads pre-existing WAR or JAR file to the Google App \
  Engine flexible environment.
 deployFlexExistingWarJarCommandName=Deploy WAR/JAR File to App Engine Flexible
+deployFlexExistingWarJarMenuLabel=Deploy WAR/JAR File to App Engine Flexible...
```

Comment:
I wonder if there's any guidance somewhere on whether to use three periods or the single ellipsis character …?

Reply:
Kit Lo, one of the IBM i18n experts [noted](https://bugs.eclipse.org/bugs/show_bug.cgi?id=230381):

> \u2026 is used in the following key in org.eclipse.birt.report.designer.core plugin. It may cause conversion problems for some langauges during translation. Using plain ASCII text "..." is recommended. Thanks!

though that was 9 years ago.  I'll see what they recommend.

Reply:
Kit reconfirmed:
> Most English .properties files should only use ASCII characters... unless you really need to use some special Unicode symbols.